src/System.IO.Pipelines/tests/PipePoolTests.cs

```diff
@@ -266,6 +267,114 @@ public async Task OnReaderCompletedCalledAfterBlocksReturned()
             pipe.Reader.Complete();
         }
 
+        [Fact]
+        public async Task WritesToArrayPoolByDefault()
```

Comment:
I guess there's nothing we can do to validate that it's actually using ArrayPool.Shared?  Just seems like a strange name for the test, given that we're not actually validating this.

Reply:
The owner being set is how we internally determine array pool or not.

Reply:
Got it.